merge.go

```diff
@@ -15,6 +15,8 @@ type MergeConfig struct {
 
 	// what it does if its matching the node
 	OnMatch *MergeSetConfig
+
+	OnMatchWithMembers *MergeSetConfigWithMembers
```

Comment:
If I'm not mistaken, it looks to allow specifying a map of members / targets, as opposed to setting a Name directly equal to a target of type `ParamString`. Is that correct?

Reply:
I wonder if it would make more sense to modify the `OnMatch` param instead of adding a new param, or at the very least consider renaming this - `OnMatchWithMembers` was not immediately clear to me

Reply:
> If I'm not mistaken, it looks to allow specifying a map of members / targets, as opposed to setting a Name directly equal to a target of type ParamString. Is that correct?

This is correct.

> I wonder if it would make more sense to modify the OnMatch param instead of adding a new param, 

Piggyback this change into `OnMatch` might destroy its existing structure, as you can see the `FunctionConfig` is highly coupled with `Member` in the `MergeSetConfig` struct. Introducing a new `Members` field into the struct might make the logic fairly convoluted.

> or at the very least consider renaming this - OnMatchWithMembers was not immediately clear to me

Agree.